src/PrestaShopBundle/Resources/config/services/adapter/group.yml

```
@@ -7,3 +7,5 @@ services:
     arguments:
       - '@prestashop.adapter.legacy.configuration'
       - '@=service("prestashop.adapter.legacy.context").getContext().language.id'
+  prestashop.adapter.group.group_data_provider:
```

Maybe an empty line above would make it easier to read 😉 
```suggestion

  prestashop.adapter.group.group_data_provider:
```